app/components/new-chat.tsx

```
@@ -101,56 +103,58 @@ export function NewChat() {
-        ></IconButton>
+        >
+        </IconButton>
         {!state?.fromHome && (
           <IconButton
             text={Locale.NewChat.NotShow}
             onClick={async () => {
               if (await showConfirm(Locale.NewChat.ConfirmNoShow)) {
                 startChat();
                 config.update(
-                  (config) => (config.dontShowMaskSplashScreen = true),
+                  config => (config.dontShowMaskSplashScreen = true),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)